modules/gitops-argo-cd-installation.adoc

```diff
@@ -23,4 +23,16 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 .. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and check *Enabled*.  
 
-. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
\ No newline at end of file
+. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
+====
+. To log in with your username and password, obtain the password for the Argo CD instance:
+.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective.
+.. Use the *Project* drop-down list and select the name of the Argo CD instance, <Argo_CD_instance_name>, in the `openshift-gitops` project.
+.. Use the left navigation panel to navigate to the *Secrets* page.
+.. Select the *openshift-gitops-cluster* instance to display the password.
+.. Copy the password.
+. Use this password and `admin` as the username to log in to the Argo CD UI in the new window.
```

Comment:
Is there a description provided for accessing the route for ArgoCD server UI? 

Please provide the preview page

Reply:
I am unsure about the description for accessing the route @varshab1210. Could you please elaborate a bit more on this? 

Preview page: https://71605--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance

Reply:
I see it in the preview page
https://71605--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance#gitops-argo-cd-installation_setting-up-argocd-instance
6th step